src/app/pages/courses-page/actions-section/actions-section.component.ts

```diff
@@ -1,15 +1,20 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+
+import { faPlusCircle, IconDefinition } from '@fortawesome/free-solid-svg-icons';
 
 @Component({
   selector: 'app-actions-section',
   templateUrl: './actions-section.component.html',
   styleUrls: [ './actions-section.component.scss' ],
 })
-export class ActionsSectionComponent implements OnInit {
+export class ActionsSectionComponent {
```

Comment:
Do we really need this `ActionsSectionComponent` wrapper ?

Reply:
I think that we can remove it. It's really a bit redundant.